mods/default/tools.lua

```diff
@@ -493,26 +493,21 @@ minetest.register_tool("default:key", {
 	end
 })
 
-minetest.register_craft({
-	type = "fuel",
-	recipe = "default:pick_wood",
-	burntime = 6,
-})
-
-minetest.register_craft({
-	type = "fuel",
-	recipe = "default:shovel_wood",
-	burntime = 4,
-})
+--
+-- Fuel
+--
 
-minetest.register_craft({
-	type = "fuel",
-	recipe = "default:axe_wood",
-	burntime = 6,
-})
+local fuel_recipes = {
```

Comment:
`wood_fuel_recipes` or `wood_tool_fuel` would be more accurate names

Reply:
Good idea, I changed it to `wood_tool_fuel` because I think that better reflects that its both tools and wood.